adalflow/adalflow/components/model_client/__init__.py

```diff
@@ -60,6 +60,10 @@
     "adalflow.components.model_client.openai_client.get_probabilities",
     OptionalPackages.OPENAI,
 )
+LMStudioClient = LazyImport(
+    "adalflow.components.model_client.lm_studio_client.LMStudioClient",
+    OptionalPackages.LMSTUDIO,
```

Comment:
The Optional package is not defined, please add it.

This dependency will also be added in the `pyproejct.toml` under /adalflow